python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1189,7 +1218,7 @@ def forward_normal(
 
         return hidden_states, residual
 
-    def forward_deepep(
+    def forward_mode_mlp_input_one(
```

Comment:
It seems that having "deepep" in the name here would be better, as it would make understanding more straightforward. For example, "forward_deepep_mode"? Also, "forward_normal" could be changed to something like "forward_tp_mode." Otherwise, users would still need to check the meanings and differences between "input_all" and "input_one."

Reply:
Well, it seems that it is no longer specific to deepep. indeed in #4836, when enable_moe_dense_fully_dp, we need to go to this "mlp_input_one" branch, even if it is a dense layer and unrelated to deepep.

Reply:
ok